lapack/testlapack/dlauu2.go

```diff
@@ -0,0 +1,112 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/exp/rand"
```

Comment:
Blank line between x/exp import and gonum imports.

Reply:
Done.